core/src/saros/session/internal/SarosSession.java

```diff
@@ -1038,6 +1042,18 @@ public void changeColor(int colorID) {
     changeColorManager.changeColorID(colorID);
   }
 
+  public void gitSendCommitRequest() {
```

Comment:
The methods do not belong here.
1. They are GIT specific and not generic VCS.
2. What do you think will happen if every new component adds method to this class ?

Reply:
> 1. They are GIT specific and not generic VCS.

The support I implement is indeed GIT specific and not generic VCS. 

> 2.What do you think will happen if every new component adds method to this class ?

Is there another/better place to add the needed methods? I will need them when I want to give the user access to the commands "send commit" and "change working directory tree".